packages/yourloops/lib/http/http.util.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { AxiosError } from 'axios'
+import httpStatus from './models/enums/http-status.enum'
+import { t } from '../language'
+import { ErrorMessageStatus } from './http.service'
+
+export class HttpUtil {
+  static handleError(error: AxiosError, excludedErrorCodes: number[] = []): Error {
+    if (!error.response || excludedErrorCodes.includes(error.response.status)) {
+      return error
+    }
+
+    if (error.response.status >= 400 && error.response.status <= 550) {
+      switch (error.response.status) {
+        case httpStatus.StatusNotFound:
+          throw Error(ErrorMessageStatus.NotFound)
```

Comment:
why the 404 is different from other in term of translation ? no use of t('error-http-404') that I would expect

Reply:
Just for reference, I simply copied/pasted this code from where it previously was so I cannot tell you why it was done this way 😄 

If something needs to be changed, I can change it but at the moment I'm not sure I see the potential impacts very clearly